### ISSUE TITLE
deepstack in dire straights?

### ISSUE THREAD

**Issue by githubDiversity** [VorlonCD/bi-aidetection#269]
have reason to believe the project is dying. I hope I am mistaken.

Given that there currently is no known reliable/working deepstack windows GPU version that I am aware of I decided to get the code and do it myself.

What I found was a surprise to me and let me just say that things need to be cleaned up real fast in order to have any life expectancy.

I have reached out to @john on github regarding the state the current code is in but havn’t heard back yet.

If there is anyone with a working windows GPU version (preferably a recent one) please zip it up and share it. I can decompile the code if need be and start from there in an effort of cleaning house.

**Comment by jgosnell56** [comment]
This is the latest version I have.
[https://www.filehosting.org/file/details/956874/DeepStack-Installer-GPU-2021.02.1.zip](url)

**Comment by githubDiversity** [comment]
thank you for your efforts @jgosnell56 I have downloaded it just to be sure.
Baswed on the filename and filesize this is the version I also tried.
What I meant was if someone can please zip up their installed C:\Deepstack folder. If that does work on my end I can then try to get the most recent source code to produce an output similar to that (only cleaned up and with the most recent code)

**Comment by jgosnell56** [comment]
I can do that as well.  Give me just a bit.

**Comment by jgosnell56** [comment]
Try this.
[https://www.filehosting.org/file/details/957003/DeepStack.zip](url)

**Comment by githubDiversity** [comment]
thanks a bunch, I'll keep one updated on how far I get.

**Comment by githubDiversity** [comment]
finally I have zeroed it down.

I can now try to compile a working 2021-6-1 windows GPU version. Are you interested and if so why? It' will help me motivate to push through

**Comment by jgosnell56** [comment]
I would certainly be interested in it. I run deep stack on a home server
with a decent NVIDIA graphics card. I have found that the GPU version of
DeepStack as far less latency than the CPU version. I assume there are many
others in the same boat or they have the same feelings.

**Comment by githubDiversity** [comment]
ugg. I am running into another wall when running a compiled windows GPU based on the latest available source code.
TL;DR; I have a lot more digging and error resolving to to before I can even begin of cleaning house. I will try but I assume the developers abandoned, if this is actually the case, it for a good reason which up until now I do not know yet.

For the tech savvy amongst us;
Process Process-2:
Traceback (most recent call last):
  File "D:\obj\windows-release\37amd64_Release\msi_python\zip_amd64\process.py", line 297, in _bootstrap
  File "D:\obj\windows-release\37amd64_Release\msi_python\zip_amd64\process.py", line 99, in run
  File "C:\DeepStack\intelligencelayer\shared\face.py", line 73, in face
    cuda=SharedOptions.CUDA_MODE,
  File "C:\DeepStack\intelligencelayer\shared\.\recognition\process.py", line 28, in __init__
    load_model(self.model, model_path)
  File "C:\DeepStack\intelligencelayer\shared\.\recognition\process.py", line 11, in load_model
    checkpoint = torch.load(path, map_location=lambda storage, loc: storage)
  File "C://DeepStack\windows_packages\torch\serialization.py", line 585, in load
    return _legacy_load(opened_file, map_location, pickle_module, **pickle_load_args)
  File "C://DeepStack\windows_packages\torch\serialization.py", line 755, in _legacy_load
    magic_number = pickle_module.load(f, **pickle_load_args)
_pickle.UnpicklingError: invalid load key, 'v'.

**Comment by Tinbum1** [comment]
I've found the developers seem to work in spurts- they go quiet and then they suddenly reappear.

**Comment by bigshow104** [comment]
Has there been any progress on this. I am using aitool and can not get it toi work with deep stack. I did have it running on a docker but am now running hte windows gpu version but still not change.

**Comment by githubDiversity** [comment]
It seems there is no more need as the developer of deepstack said that a compiled  windows GPU version will now be included in the release cycles
